demos/common/export_models/export_model.py

```diff
@@ -287,13 +298,17 @@ def export_embeddings_model(model_repository_path, source_model, model_name, pre
                 if os.system(optimum_command):
                     raise ValueError("Failed to export embeddings model", source_model)
                 set_rt_info(tmpdirname, 'openvino_model.xml', 'config.json')
+                if truncate:
+                    max_context_length = get_models_max_context(tmpdirname, 'config.json')
+                    if max_context_length is not None:
+                        set_max_context_length = "--max_length " + str(get_models_max_context(tmpdirname, 'config.json'))
                 os.makedirs(embeddings_path, exist_ok=True)
                 shutil.move(os.path.join(tmpdirname, 'openvino_model.xml'), os.path.join(embeddings_path, 'model.xml'))
                 shutil.move(os.path.join(tmpdirname, 'openvino_model.bin'), os.path.join(embeddings_path, 'model.bin'))
             tokenizer_path = os.path.join(model_repository_path, model_name,'tokenizer', version)
             print("Exporting tokenizer to ", tokenizer_path)
             if not os.path.isdir(tokenizer_path) or args['overwrite_models']:
-                convert_tokenizer_command = "convert_tokenizer -o {} {}".format(tmpdirname, source_model) 
+                convert_tokenizer_command = "convert_tokenizer -o {} {} {}".format(tmpdirname, source_model, set_max_context_length) 
```

Comment:
Can we pass this option via optimum cli? 
This command has been removed for text generation and maybe in the future we would also like to remove it for other endpoints.

Reply:
@mzegla  Maybe in the future it could be defined in runtime. To be decided for next release.